sdk/identity/azure-identity/README.md

```diff
@@ -70,6 +66,10 @@ this library's credential classes.
 
 `DefaultAzureCredential` is appropriate for most applications which will run in the Azure Cloud because it combines common production credentials with development credentials. `DefaultAzureCredential` attempts to authenticate via the following mechanisms in this order, stopping when one succeeds:
 
+>Note: `DefaultAzureCredential` is intended to simplify getting started with the SDK by handling common
```

Comment:
"SDK" refers to the collection of libraries across the languages. When referring to a single language's library, as is the case here, use "library".
```suggestion
>Note: `DefaultAzureCredential` is intended to simplify getting started with the library by handling common
```